### DISCUSSION TITLE
Hide Map Buttons

### DISCUSSION THREAD

**Discussion by MotoUKRider** [devemux86/cruiser#185] [Q&A] [answered]
In some situations I've noticed that the "hide map buttons timer" does not "fire" - definitely when you import a route and start navigating but *without* touching the map.  It seems that, from app startup, the timer only takes effect when you touch the map.

I only say this because I've set a route up more than once, put my (non-touch-friendly) gloves on and been presented with the buttons on my screen during navigation!

On a completely unrelated note, I now carry a rubber-tipped stylus pen in my tank-bag :)

**Comment by devemux86** [comment]
@MotoUKRider thanks for the report!

Indeed the buttons auto hide after the first touch of the map.
I will note it to see how we can improve it. :slightly_smiling_face:

**Comment by devemux86** [comment]
> It seems that, from app startup, the timer only takes effect when you touch the map.

It will be improved in next Cruiser: on app start, buttons will be hidden if the setting is enabled.

**Comment by devemux86** [comment]
Also we probably need something extra for the zoom buttons?

Now they always auto hide, but in navigation some users may prefer the zoom buttons on screen.

**Reply by DolomitesTom** [reply]
Maybe, but for me auto hide is good

**Comment by devemux86** [comment, marked as answer]
Auto hide of map buttons is improved in [Cruiser 3.0.22](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
